apis/duck/v1alpha1/addressable_types.go

```diff
@@ -26,15 +26,18 @@ import (
 
 // Addressable provides a generic mechanism for a custom resource
 // definition to indicate a destination for message delivery.
-// (Currently, only hostname is supported, and HTTP is implied. In the
-// future, additional schemes may be supported, and path components
-// ala UI may also be supported.)
-
+// Hostname is deprecated as of v0.6 and replaced with URL. It will be removed in ~v0.7
```

Comment:
We should avoid removing things until Beta
```suggestion
// Hostname is deprecated as of v0.6 and replaced with URL.
```

Reply:
I'll change the comment to "It will be removed in v0.8 or later", which gives the guarantee that it will be supported at least one version after it has been deprecated. ie deprecate in 0.6, supported in 0.7, 0.8+ no guarantee. We may decide to keep this till Beta or remove it if we find it easier to do early.

Reply:
We haven't removed any fields from our API to date.  I've been thinking about this myself because CRD versioning is so far out.  I will try to put forward a proposal soon.

Reply:
Sounds good. Meanwhile I'll keep the new verbiage which guarantees one version support, after that no guarantees. Add me to your proposal reviewer list when you have it.